flow/cmd/handler.go

```diff
@@ -129,6 +129,9 @@ func (h *FlowRequestHandler) CreateCDCFlow(
 	workflowOptions := client.StartWorkflowOptions{
 		ID:        workflowID,
 		TaskQueue: h.peerflowTaskQueueID,
+		SearchAttributes: map[string]interface{}{
+			"MirrorName": cfg.FlowJobName,
```

Comment:
move this to a constant

const MirrorNameSearchAttribute = ...

and use this constant across files